### PR TITLE
121 multiple location types

### LINKED ISSUE #121: Allow input of multiple location types and physical types
Source: https://github.com/onaio/fhir-tooling/pull/120#issuecomment-1920701785

Initially the importer tool limited the location type and physical type to only **Buildings** and **Jurisdicitons**

This needs to change to allow more options:
PhysicalType - https://hl7.org/fhir/R4B/valueset-location-physical-type.html
Type - https://terminology.hl7.org/3.1.0/ValueSet-v3-ServiceDeliveryLocationRoleType.html

### PR DESCRIPTION
**Pull request by TraciebelWairimu**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #121

**Engineer Checklist**
- [ ] I have run `./gradlew spotlessApply` to check my code follows the project's style guide
- [ ] I have built and run the efsity jar to verify my change fixes the issue and/or does not break the application

### CODE REVIEW

**Comment by TraciebelWairimu** [comment]
Before incase there was an empty cell and no location type or physical type was input, the variables "$t_code" or "$pt_display" were passed, I have changed it to delete the columns incase no values are passed.

**Comment by Wambere** [comment]
@TraciebelWairimu let's update the documentation for this as well

File: importer/main.py [hunk 1]
```diff
@@ -250,24 +245,17 @@ def location_extras(resource, payload_string):
...
-            )
-        elif physicalType == "jurisdiction":
-            payload_string = payload_string.replace("$pt_code", "jdn").replace(
-                "$pt_display", "Jurisdiction"
-            )
+        if len(physicalType.strip()) > 0:
+            payload_string = payload_string.replace("$pt_display", physicalType)
+        if len(physicalTypeCode.strip()) > 0:
+            payload_string = payload_string.replace("$pt_code", physicalTypeCode)
         else:
```
Comment (Wambere) [inline importer/main.py:252]: This else would only happen if the physicalTypeCode is missing but not if the physicalType is missing
Wondering is that an ok assumption to make? That whoever is filling in the csv will always fill in for both in cases that need filling in?

**Review by pld — APPROVED** [review]

File: importer/main.py (hunk 1 above)
Comment (dubdabasoduba) [reply importer/main.py:252]: @Wambere  yes, the assumption is right.

**Review by Wambere — APPROVED** [review]

**Pull request merged**